src/data.rs

```diff
@@ -0,0 +1,272 @@
+//! Custom structs and enums for mac-notification-sys.
```

Comment:
This could be renamed to notification.rs because it contains no actual data but structs for the actual notification